src/intentManager.js

```diff
@@ -61,6 +61,14 @@ define([
         return widget;
     }
 
+    function mediaWidget(mug, options) {
```

Comment:
Should this be called `printTemplateWidget` since it already inherits from a `media` widget?